github/ci/services/prometheus-stack/manifests/grafana.yaml

```diff
@@ -141,7 +141,7 @@ data:
     provisioning = /etc/grafana/provisioning
     [auth.anonymous]
     enabled = true
-    org_name = KubeVirt community
+    org_name = Main Org.
```

Comment:
This looks odd?

Reply:
Yep, but it is the value of the default org created, without it there's an "unknown org" error in the logs and people not already logged get to the logging screen (not direct access given that they can be assigned to a default org). This is the same value that we use in the existing deployment https://github.com/kubevirt/project-infra/blob/master/github/ci/prometheus/templates/grafana-config.yaml#L16